CMakeLists.txt

```diff
@@ -552,7 +552,7 @@ endif()
     add_test(NAME test_OFDM_modem_700E_AWGN
              COMMAND sh -c "cd ${CMAKE_CURRENT_BINARY_DIR}/src;
                             ./ofdm_mod --in /dev/zero --testframes 10 --ldpc --mode 700E |
-                            ./cohpsk_ch - - -22 --Fs 8000 |
+                            ./ch - - --No -22 |
```

Comment:
`$<TARGET_FILE:ch>`, `$<TARGET_FILE:ofdm_mod>` and `$<TARGET_FILE:ofdm_demod>`.

Reply:
Similar comment for the rest of this file.

Reply:
As per above the `cd ${CMAKE_CURRENT_BINARY_DIR}/src;` puts in the correct directory so the following code works.  It should be "build dir" agnostic - we've been writing/running tests like this for a while now, although some earlier tests still use the `$<TARGET_FILE:ch>` style.

Reply:
Yep, no worries. 👍